pom.xml

```diff
@@ -11,7 +11,7 @@
 
   <groupId>org.jenkins-ci.plugins</groupId>
   <artifactId>structs-parent</artifactId>
-  <version>${revision}${changelist}</version>
+  <version>${changelist}</version>
   <packaging>pom</packaging>
 
   <name>Structs: Parent</name>
```

Comment:
Best to do https://github.com/jenkinsci/structs-plugin/pull/112#discussion_r754444957 first.

Reply:
It is _possible_ to do CD on a multimodule project. RPU needs to have `cd.enabled` set, IIRC, on at least one module. More complicated than you really need here. Just make sure RPU lists only the plugin itself (not the parent POM nor the annotation), then try to do this.

Reply:
#114

Reply:
Yikes, looks like I opened up a bigger can of worms than I expected. I agree this is probably more complicated than we need. I am fine with not proceeding, but seeing #114, is it worth it to continue? It looks like only the directory structure is changed and nothing else. So it shouldn't affect any plugins that are depending on this?

Reply:
> I am fine with not proceeding

I just meant merge #114 first, then adapt this to match (so there would be just one POM edit, and it would be to the plugin POM).

> It looks like only the directory structure is changed and nothing else. So it shouldn't affect any plugins that are depending on this?

In #114? Yes, no change to artifacts, other than `structs-parent.pom` no longer being deployed.